abakus-statutter/08-generalforsamling.tex

```diff
@@ -94,12 +94,11 @@ \subsubsection{}
 \subsection{Valg}
 \subsubsection{}
 Samtlige stillinger i Hovedstyret velges av Generalforsamling ved
-alminnelig flertall. Så lenge ingen kandidater oppnår alminnelig flertall
-fjernes den kandidaten med lavest antall stemmer og det stemmes på nytt. Blanke
+alminnelig flertall. Valget avholdes med Preferansevalg (jfr. §0.6). Blanke
 stemmer teller og kan underkjenne kandidatene.
 
 \subsubsection{}
-Hvis det til slutt ikke gjenstår noen kandidater kan generalforsamlingen
+Hvis ingen kandidater oppnår alminnelig flertall kan generalforsamlingen
```

Comment:
Whenever one speaks of _the_ Generalforsamling, the "G" is supposed to be capital. Whenever one speaks of the event generalforsamling, the "G" is supposed to be a lower case one.